### ISSUE TITLE
`:binding` options do not apply when the form has empty body

### ISSUE THREAD

**Issue by huahaiy** [kkinnear/zprint#73]
For example,  for
```
(let [a :a b :b c "really really long string"])
```
The options `:binding {:force-nl? true :flow? true :indent 0 :nl-separator? true}` have no effect at all.

But if the form is 
```
(let [a :a b :b c "really really long string"] a)
```
instead, the options apply

**Comment by kkinnear** [comment]
Yes, that was by design.  I would guess that you would like the binding options to apply even if there is no body, or you wouldn't have filed an issue.  I'll see what I can do about changing it to consider a two-element list a `:binding` form.

**Comment by huahaiy** [comment]
Thanks a lot. It is indeed important for some DSLs that use this kind of two-element binding forms extensively.

**Comment by kkinnear** [comment]
I have implemented this, and it will be in the next release, which will probably be early next year.  The rest of this year is looking pretty busy.  I'll close this issue when I release it so you will know.

**Comment by huahaiy** [comment]
Great! Thanks. Have a great holidays!

**Comment by kkinnear** [comment]
Fixed in 0.4.14.  Thanks for reporting this.
